engine/src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -40,6 +40,7 @@ case class WorkflowDescriptor(id: WorkflowId,
                               backend: Backend,
                               configCallCaching: Boolean,
                               lookupDockerHash: Boolean,
+                              configWorkflowFailureMode: WorkflowFailureMode,
```

Comment:
This is actually in a slightly weird spot w.r.t validation, defaults, etc. I'm going to wait until @gauravs90 commits the ValidationActor stuff because it's all going to be so much neater in the brave new world.